modules/clipboard/init.zsh

```diff
@@ -0,0 +1,62 @@
+#
+# Integrates the zsh clipboard with the system clipboard.
+#
+# Authors:
+#   Nir Friedman <quicknir@gmail.com>
+#
+
+# If running Mac OS, set this to 1
+local IS_MAC_OS=0
```

Comment:
You don't need that. Check for `[[ "$OSTYPE" == darwin* ]]`. See the [utility](https://github.com/sorin-ionescu/prezto/blob/master/modules/utility/init.zsh#L114) module for an example.